README.md

```diff
@@ -246,6 +246,6 @@ and will automatically be started for subsequent system boots.
 ## Issues
 Please use the [GitHub integrated issue tracker](https://github.com/XaF/TraktForVLC/issues) for every problem you can encounter. Please **DO NOT** use my email for issues or walkthrough.
 
-When submitting an issue, please submit a TraktForVLC logfile showing the error. You can start TraktForVLC in debug mode (```--debug``` option) to obtain more thorough logs and with small timers (```--small-timers``` option) to speed up its internal loop. The resulting logfile (```TraktForVLC-DEBUG.log```) should then be available in your ```logs/``` directory.
+When submitting an issue, please submit a TraktForVLC logfile showing the errors you are running into. You can run TraktForVLC in debug mode (```--debug``` option) for more thorough logs when debugging manually. To save these contents into a separate logfile (```TraktForVLC-DEBUG.log```) in your ```logs/``` directory, use the ```--daemon``` option or ```--pidfile some_file.txt``` when running the script. You can additionally use the option ```--small-timers``` to speed up the script's internal loop.
```

Comment:
Can you merge these changes with the other commit ?

Reply:
Which one? (Sorry, this is showing as outdated, so didn't see the comment right away and I'm confused now, ha.)

Reply:
Oh wait that's my bad, I did not see it was outdated and rewritten in the other commit :)